### DISCUSSION TITLE
MINIO_ERASURE_SET_DRIVE_COUNT default

### DISCUSSION THREAD

**Discussion by zhl19911203** [minio/minio#20311] [Q&A] [answered]
What is the default value of the MINIO_ERASURE_SET_DRIVE_COUNT parameter, if it is automatically calculated, how can I check this value after the deployment is completed, whether to use the command or other way to view, because the configuration ratio of the EC has something to do with this

**Comment by marktheunissen** [comment]
It is auto calculated, and you can see the value as follows in the output under `Erasure stripe size`:

```
 ❯ mc admin info play
●  play.min.io
   Uptime: 13 hours
   Version: 2024-08-22T09:43:48Z
   Network: 1/1 OK
   Drives: 4/4 OK
   Pool: 1

┌──────┬───────────────────────┬─────────────────────┬──────────────┐
│ Pool │ Drives Usage          │ Erasure stripe size │ Erasure sets │
│ 1st  │ 27.0% (total: 80 GiB) │ 4                   │ 1            │
└──────┴───────────────────────┴─────────────────────┴──────────────┘

21 GiB Used, 1,214 Buckets, 7,277 Objects, 209 Versions, 43 Delete Markers
4 drives online, 0 drives offline, EC:2
```

https://min.io/docs/minio/linux/reference/minio-server/settings/core.html#envvar.MINIO_ERASURE_SET_DRIVE_COUNT

**Reply by zhl19911203** [reply]
Thanks for the answer
Erasure Coding What are the calculation rules？
Because I want to evaluate the storage ratio based on the number of disks, I need to know the Erasure stripe size in advance

**Reply by klauspost** [reply, marked as answer]
https://min.io/product/erasure-code-calculator
